report/metrics_handles.go

```diff
@@ -38,15 +38,15 @@ var (
 func SetupWindowsHandlesMetrics(logger *logp.Logger, reg *monitoring.Registry) {
 	beatProcessSysInfo, err := sysinfo.Self()
 	if err != nil {
-		logger.Error("Error while getting own process info: %v", err)
+		logger.Errorf("Error while getting own process info: %s", err)
 		logger.Error(fileHandlesNotReported)
 		return
 	}
 
 	var ok bool
 	handleCounter, ok = beatProcessSysInfo.(types.OpenHandleCounter)
 	if !ok {
-		logger.Error("Process does not implement types.OpenHandleCounter: %v", beatProcessSysInfo)
+		logger.Errorf("Process does not implement types.OpenHandleCounter: %v", beatProcessSysInfo)
```

Comment:
For consistency this could also be `%s`. Also `%s` usually produces an output  that's more log friently.
```suggestion
		logger.Errorf("Process does not implement types.OpenHandleCounter: %s", beatProcessSysInfo)
```